DataFormats/HcalDigi/src/classes.h

```diff
@@ -16,7 +16,8 @@
 
 // dummy structs to ensure backward compatibility
 struct HcalUpgradeDataFrame {
-  typedef HcalDetId key_type;
+  using key_type = HcalDetId;
+  HcalDetId id() const { return HcalDetId(); }
```

Comment:
This is needed to compile `edm::Wrapper<edm::SortedCollection<HcalUpgradeDataFrame>>` that is requested by `classes_def.xml`.